src/libs/notification/tests/CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ if (BUILD_TESTING)
 		add_test (NAME ${name}
```

Comment:
@waht Isn't it easily possible to use do_test here?

@ingwinlu Feel free to get rid of add_test duplicates so that we do not have set_property scattered around everywhere.